tools/cldr-apps-webdriver/scripts/selenium-grid-start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+VER=4.32.0
+URL=https://github.com/SeleniumHQ/selenium/releases/download/selenium-${VER}/selenium-server-${VER}.jar
+FILE=selenium-server.jar
```

Comment:
I deleted the jar file that was previously checked into the repo.

This script isn't used within docker - we spin up a standalone chrome browser in a container.